library/ecp_invasive.h

```diff
@@ -31,6 +31,14 @@
 #include "bignum_mod.h"
 #include "mbedtls/ecp.h"
 
+/*
+ * Turning this option on enables using the new bignum code in the ECC modules.
+ *
+ * WARNING: ECC implementation using the new bignum code is a work in progress,
+ * this option serves only development and testing purposes.
+ */
+//#define MBEDTLS_ECP_WITH_MPI_UINT
```

Comment:
That's a much more robust approach than fiddling with `CFLAGS`. But why not put the option in `mbedtls_config.h` and completely follow the standard process? (And add it to the list of exceptions to the `full` config in `scripts/config.py`, with the justification that it's incompatible with the normal ECP.)

Reply:
I felt that this is much less mature and maybe bigger than most of the features we mark as experimental in `mbedtls_config.h`. But I don't have strong opinion on this and happy to put it in `mbedtls_config.h` if you think that it is better like that.

Reply:
We had `MBEDTLS_USE_PSA_CRYPTO` as experimental for a long time, it doesn't get much bigger and more invasive than that.